census/estimator/README.md

```diff
@@ -0,0 +1,135 @@
+# Census: TF.Learn Distributed Training and Prediction
+
+This sample walks you through training and prediction, including distributed training, and hyperparameter tuning.
+
+All commands below assume you have cloned this repository and changed to the currend directory. It also assumes you have:
+
+1. Installed the the [Google Cloud SDK](https://cloud.google.com/sdk)
+2. (For Cloud Training) [Set up the Google Cloud Machine Learning API](https://cloud.google.com/ml/docs/how-tos/getting-set-up) NOTE: You can safely ignore the "Setting Up Your Environment" section. 
+
+## Local Training
+
+### Getting Data
+
+Throughout we will use a census dataset for training. For local training, you can read them directly from Google Cloud Storage or if you have an unreliable network, can download them to your local file system.
+
+```
+TRAIN_FILE=gs://cloudml-public/census/data/adult.data.csv
+EVAL_FILE=gs://cloudml-public/census/data/adult.test.csv
+```
+
+To use locally simply copy them down with gsutil:
+```
+mkdir data
+gsutil cp gs://tf-ml-workshop/widendeep/* data/
+```
+
+And use the local locations whereever the environment variables above are used.
+
+
+### Installing Dependencies
+
+The local training environment for Cloud ML uses your local python installation. So you will need to have Python 2.7 and install TensorFlow following the instructions [here](https://www.tensorflow.org/get_started/os_setup).
+
+### Running Training
+
+Then you can run training with the following command:
+
+```
+gcloud beta ml local train \
+    --module-name trainer.task \
+    --package-path trainer/ \
+    -- \
+    --train-files $TRAIN_FILE \
+    --eval-files $EVAL_FILE \
+    --train-steps 1000 \
+    --job-dir output
```

Comment:
No. This is actually passed to the service, which validates the location which is useful functionality. Additionally in hyperparam tuning it will extend the job-dir with a trial ID which is required.